### ISSUE TITLE
still some problem with graves :/

### ISSUE THREAD

**Issue by lightning02** [OpenMods/OpenBlocks#789]
Well the problem is that i was killed by a Remombra from lycanites creature in Silkglades biome from  biomes o'plenty on a simply vanilla dirt and my tombstone was not generate.
In the server log the gravestone was said to be generate on coordinates (-251,80,-23)
 but when i relogged and i check for my grave it was not there nor underneath. 
After i found in the log the command to restore my grave and i've use it. It can be restored with command '/ob_inventory restore _Red_Lightning_ _Red_Lightning_-2016-07-28_22.10.23-death-0' 
This is the log http://pastebin.com/8pZeM0Tu
And this is the place where i died. http://imgur.com/Rjxqbfq
Normally the grave generates well.
The problem with enviromine was solved at least a week ago..

**Comment by boq** [comment]
Hmm, looks like grave block was already placed (successfully) and filed with items, so something has happened after that. Maybe block was destroyed later? OpenBlocks logs when graves are destroyed, but you would need to change log level to "DEBUG" (not sure how to do that with your setup).

**Comment by lightning02** [comment]
Maybe I can enable debug mode in openblocks config file? Is there a option for that?

**Comment by boq** [comment]
No, it's probably something in server settings. Or maybe separate log file.
